### PR TITLE
AG-8466 -  Convert Enterprise Crosshairs Plunkers to examples

### PR DESCRIPTION
**Pull request by manapeirov**
PR for https://ag-grid.atlassian.net/browse/AG-8466

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-crosshairs/examples/crosshair-custom-label/data.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// Source: https://exoplanetarchive.ipac.caltech.edu/
```
Comment (lsjroberts) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-crosshairs/examples/crosshair-custom-label/data.ts:1]: This source doesn't look related?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-crosshairs/examples/crosshair-default-label-custom-css/data.ts [hunk 2]
```diff
@@ -0,0 +1,5741 @@
+// Source: https://exoplanetarchive.ipac.caltech.edu/
+export function getData() {
```
Comment (lsjroberts) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-crosshairs/examples/crosshair-default-label-custom-css/data.ts:2]: this seems excessive?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-crosshairs/examples/enabling-crosshairs/data.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+// Source: https://exoplanetarchive.ipac.caltech.edu/
```
Comment (lsjroberts) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-crosshairs/examples/enabling-crosshairs/data.ts:1]: Unrelated source

File: charts-enterprise-modules/ag-charts-enterprise/src/main.ts [hunk 4]
```diff
@@ -65,4 +73,10 @@ export class AgEnterpriseCharts {
 
         return AgChart.create(options as any);
     }
+
+    public static update(chart: AgChartInstance, options: AgChartOptions) {
+        new LicenseManager(options.container as any).validateLicense();
+
```
Comment (alantreadway) [inline charts-enterprise-modules/ag-charts-enterprise/src/main.ts:79]: Nit: We probably don't need this on `update()`, as we'll spam the console with errors on every update?

**Review by alantreadway — APPROVED** [review]
LGTM
